lib/css/components/_stacks-widget-static.less

```diff
@@ -28,7 +28,7 @@
     font-size: @fs-body1;
     background-color: var(--white);
 
-    &:not(.s-anchors) a:not(.button):not(.s-btn):not(.post-tag):not(.s-sidebarwidget--action) {
+    &:not(.s-anchors) a:not(.button):not(.s-btn):not(.post-tag):not(.s-sidebarwidget--action):not(.s-user-card--link ) {
```

Comment:
I think our sidebar widget is a bit too opinionated here. These overrides seem like an anti-pattern. Not your problem 🤗 Noting it for me in the future.